app/src/pages/trace/TracePage.tsx

```diff
@@ -1426,7 +1426,7 @@ function SpanEventsList({ events }: { events: Span["events"] }) {
         return (
           <ListItem key={idx}>
             <Flex direction="row" alignItems="center" gap="size-100">
-              <View>
+              <View overflow="visible">
```

Comment:
```suggestion
              <View flex="none">
```

Reply:
@axiomofjoy typically you don't want things to overflow if you don't want things scrolling. The only case you want to display things overflowing are things like tooltips. In this case an explicit layout is desired where the icon takes up allocated space without it's size getting compromised by a sibling's width. To do this you tell the flex layout that the element shouldn't collapse no matter the consequence. https://css-tricks.com/snippets/css/a-guide-to-flexbox/#aa-flex

Reply:
Thanks!